### ISSUE TITLE
[REQ] Costal Crew

### ISSUE THREAD

**Issue by Winblackka** [Jackett/Jackett#10715]
German private ALT-Tracker

https://coastal-crew.bounceme.net/

I would like to get this one added to Jackett. 

Thank you!

**Comment by Winblackka** [comment]
Do you need some logs or something to get this done?

Thank you!

**Comment by ilike2burnthing** [comment]
I've registered for an account. No email as of yet, and trying to log in fails. I'll give it a few hours and then message them.

Note to self: see [dragonworldreloaded.yml](https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Definitions/dragonworldreloaded.yml)

**Comment by Winblackka** [comment]
Okay, thx for your reply.

> I've registered for an account. No email as of yet, and trying to log in fails. I'll give it a few hours and then message them.
> 
> Note to self: see [dragonworldreloaded.yml](https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Definitions/dragonworldreloaded.yml)

**Comment by ilike2burnthing** [comment]
[v0.17.231](https://github.com/Jackett/Jackett/releases/tag/v0.17.231)

**Comment by ilike2burnthing** [comment]
https://coastal-crew.bounceme.net/

```text
Ihr müsst euch alle neu registrieren!!!
Im Spamordner nachschaun für Bestätigungsmail!!!
```

```text
‎You all have to re-register!!! ‎‎ ‎
‎‎‎Check the spam folder for confirmation mail!!! ‎
```

Reregistered, yet to get an email, no way to reset password.

**Comment by garfield69** [comment]
Me too. Used my old credentials to register but did not get an email. 
Another instance of gmail blocking?

**Comment by garfield69** [comment]
I switched to my protonmail and have created an account
When I got the confirmation email ans used the link I got a message that the account was already activated.
But when I went to login I got flagged as having a double IP.
When I used hideme and tried to login then I got the Error : wrong username, password or PIN, the same error that I got with my first registration/login attempt.
So I give up, I cannot be bothered trying again.

**Comment by garfield69** [comment]
But using hideme and my first registration I have logged in %-/
So @ilike2burnthing  try to use your recent registration credentials and see if the work for you.
If they don't, use a proxy and try again.

**Comment by garfield69** [comment]
Admin have spotted the double ip and queried who is my other a/c. I might me in trouble ;-(

**Comment by garfield69** [comment]
Admin have delete the other a/c but left me with the first one so all good for now.

**Comment by MrSliff** [comment]
> Admin have delete the other a/c but left me with the first one so all good for now.

Ok, so everything fine now, that you can figure out to change to the new login form according to [#13193](https://github.com/Jackett/Jackett/issues/13193)? I would have said, maybe check spam folder, but seems you figured how to login.

**Comment by garfield69** [comment]
yes either @ilike2burnthing  or I will work on getting the indexer up to date when we next have the free time. I'm going to bed.

**Comment by MrSliff** [comment]
Ok, thanks. Appreciate your work!

**Comment by ilike2burnthing** [comment]
I'm in [insert generic hacking gif]

**Comment by garfield69** [comment]
v0.20.959

**Comment by MrSliff** [comment]
Me again. Thanks for your work. It seems the login was not the only problem. When testting the tracker i get this error:

` An error occurred while testing this indexer
Test search in Coastal-Crew => Found no results while trying to browse this tracker`

**Comment by ilike2burnthing** [comment]
Working fine here, and the whole indexer was updated not just the login.

Reconfigure the indexer and see if that resolves your issue. If not, but you can access https://coastal-crew.bounceme.net/browse.php in browser from at least the same IP and using the same DNS (ideally from the same device too), then open a new issue and provide the information requested.

**Comment by ilike2burnthing** [comment]
As discovered by garfield69, by default the tracker will just show a red X for all results instead of a download button. The tracker requires that you set your `seed time` before it will even display a download link.

To resolve this:
https://coastal-crew.bounceme.net/my.php > Meine Seedzeit > set your seed times or tick that your seed 24hrs a day

**Comment by garfield69** [comment]
v0.22.1691
